pootle/apps/pootle_app/management/commands/calculate_checks.py

```diff
@@ -36,87 +33,16 @@ class Command(PootleCommand):
         ),
     )
     option_list = PootleCommand.option_list + shared_option_list
-
-    cached_methods = [CachedMethods.CHECKS]
     process_disabled_projects = True
 
     def handle_all_stores(self, translation_project, **options):
-        calculate_checks(check_names=options['check_names'],
-                         translation_project=translation_project)
+        QualityCheckUpdater(
+            options['check_names'],
+            translation_project=translation_project)
 
     def handle_all(self, **options):
         if not self.projects and not self.languages:
             logging.info(u"Running %s (noargs)", self.name)
-
-            try:
-                calculate_checks(check_names=options['check_names'])
-            except Exception:
-                logging.exception(u"Failed to run %s", self.name)
+            QualityCheckUpdater(options['check_names']).update()
         else:
             super(Command, self).handle_all(**options)
-
-
-def calculate_checks(check_names=None, translation_project=None):
-    store_fk_filter = {}
-    unit_fk_filter = {}
-
-    if translation_project is not None:
-        store_fk_filter = {
-            'store__translation_project': translation_project,
-        }
-        unit_fk_filter = {
-            'unit__store__translation_project': translation_project,
-        }
-
-    logging.info('Calculating quality checks for all units...')
```

Comment:
This was lost, therefore there might not be any feedback on the command being run in some scenarios.